### PR TITLE
refs #233: added look_at_point property with migration

### PR DESCRIPTION
**Pull request by drbbgh**
@mapedorr please review the code, especially the migration manager changes.

### CODE REVIEW

**Review by mapedorr — APPROVED** [review]
I think you guys made a great job implementing this. I'm gonna make some tests in order to check everything is working, and after that, if nothing breaks, we'll be ready to merge this.

File: addons/popochiu/migration/migrations_manager.gd
[the remark this answers is not in the document]
Comment (drbbgh) [reply addons/popochiu/migration/migrations_manager.gd:1]: I'm not sure either, but we had to do this because starting with a project that had undergone migration 2, so that `popochiu_data.cfg` contained:
```
[last_migration]

version=2
```
we ended up with:
```
[last_migration]

version=5
```
after running migration 3.

[the remark this answers is not in the document]
Comment (stickgrinder) [reply addons/popochiu/migration/migrations_manager.gd:1]: @mapedorr maybe the removal was too severe, and the logic still needs to keep a counter of the migrations that way, but if you look at it, you discover the original one is actually wrong.

To spell it in plain English, every time you see a migration, you check its version is below the one saved in the config file. If it is, you add it to the config file.

Why?

Maybe you want to have a local variable that starts at zero, increment that one, and once you have reached the point where a migration is needed, you write that final value to the config file.

But since the final version is always written into the config file with the last migration run, I see no reason why you must account for that.

If you can come up with the original use-case, we can try and understand what's the right way to do that.

[the remark this answers is not in the document]
Comment (mapedorr) [reply addons/popochiu/migration/migrations_manager.gd:1]: We can leave this as it is right now. I can't remember the original use-case appart from what I already mentioned, and probably no one has needed to use the migrations until now (the only one was Papernoise, but he was using them before we released version 2), so the change won't affect too much users.

**Review by mapedorr — APPROVED** [review]
Making the last tests.
